kong/request.go

```diff
@@ -51,3 +46,12 @@ func (c *Client) NewRequest(method, endpoint string, qs interface{},
 	}
 	return req, nil
 }
+
+// NewRequest creates a request based on the inputs.
+// endpoint should be relative to the baseURL specified during
+// client creation.
+// body is always marshaled into JSON.
+func (c *Client) NewRequest(method, endpoint string, qs interface{},
+	body interface{}) (*http.Request, error) {
```

Comment:
Is this a bit more descriptive?
```suggestion
func (c *Client) NewRequest(method, relativePath string, qs interface{},
	body interface{}) (*http.Request, error) {
```